### DISCUSSION TITLE
post ai.dial报错了Invalid request or non-existant command

### DISCUSSION THREAD

**Discussion by fengyh13** [rts-cn/rts#32] [XSwitch]
我使用nodejs请求AIAPI,请求参数是： got  post  application/json
{
  jsonrpc: '2.0',
  method: 'ai.dial',
  id: 1706280989320,
  params: {
    application: 'ai',
    dial_string: "{leg_timeout=30,execute_on_answer='sched_hangup +10'}user/1004",
    from: '1005',
    url: 'http://xxx:1233',
    external_tracking_id: '1706280989320',
    private_data: { param1: '1', seq: '2' }
  }
}
返回错误信息：
{
  "jsonrpc": "2.0",
  "error": {
    "code": -32600,
    "message": "Invalid request or non-existant command"
  },
  "id": 1706280989320
}
调试的阿里的TTS/ASR都已经通了，http请求报错了，请问是哪里错了

**Comment by seven1240** [comment]
模块加载了没有？load mod_ai
